src/pages/open-source.js

```diff
@@ -38,6 +52,20 @@ const OpenSourcePage = props => (
           seoTitle
           seoDescription
           statement
+          talksSectionImage {
+            title
+            file {
+              url
+            }
+          }
+          talksSectionTitle
+          talksSectionTalks {
+            title
+            type
```

Comment:
We don't need this field. The type should always be 'talk'.

_But_, since we can't force Marketing to enter `ExternalResource`s which are only of type 'talk', we _could_ use the type field to make a check here in our codebase. Since you're pulling the type out here anyway, it might be an idea to do this check?

Reply:
True. I was also thinking about that and then forgot about it. Since we only have 1 type at the moment it's not really necessary I'd say. As soon as we would get at least 2, we could. What do you think?

Reply:
When you use the `ExternalResource` content type, in Contentful, this is the view:
![image](https://user-images.githubusercontent.com/15656538/58807744-95a92600-8610-11e9-93f4-36ca04a00b44.png)

Here is one where the `type` is talk:
![image](https://user-images.githubusercontent.com/15656538/58807789-a9548c80-8610-11e9-86ed-7bfd60a34c76.png)

But there are others where the `type` is something different:
![image](https://user-images.githubusercontent.com/15656538/58807887-dacd5800-8610-11e9-90bf-8e75c48f1a37.png)

Here is the place in Contentful where we attach the "Talks" to the open source page - this requires Marketing to enter an _External Resource_ but we have no way of restricting Marketing to only put in External Resources _of type: 'talk'_.

So, knowing that someone could accidentally put some other type of external resource in here, I reckon we should check in the codebase & strip out anything we're getting from Contentful which is not of type: 'talk'.

Do you see what I mean?

Reply:
Oh. I see. Ok, in that case I will add a safety net so we only get `talks`. Thanks for the detailed explanation and screenshots